### ISSUE TITLE
Wizbang decklist issue

### ISSUE THREAD

**Issue by Kummo33** [HearthSim/Hearthstone-Deck-Tracker#3944]
When I play with the new wizbang deck from Saviors of Uldum. The cards keep disappearing and appearing like they do with the Shadowcloak decks. 

I remember when after I got the latest version, the new wizbang deck names were already in the deck list, however i deleted all the decks so I could get new stats and so old wizbang decks don't interfere with the old ones.  

I thought the new wizbang decks would regenerate, but it doesn't seem to be the case. Is there any way to get them back?

**Comment by howiieb** [comment]
This issue was temporary in the transition between expansions, when Deck Recipes are updated ingame for the expansion but Whizbang is still pulling old recipes on the server side. This [has been a closed issue before](https://github.com/HearthSim/Hearthstone-Deck-Tracker/issues/3831).

**Comment by Kummo33** [comment]
Yeah I saw the previous posts about it when searching for a fix. But I remember I saw new deck list names for wizbang like the obelisk deck already inside when I started it so I thought I screwed up somehow by deleting the decks and it didn't regenerate. Is this not the case?

**Comment by howiieb** [comment]
The new deck recipes are added to the game as soon as the expansion update is released (so, before the actual release of the expansion). However, this doesn't mean Whizbang will let you play with the new cards early - when you queue with Whizbang, it randomly selects one of the _old_ recipes from last expansion on the server side. The server side has the old recipes, your client doesn't, therefore the issue occurs. With the expansion out now, Whizbang is working again.

**Comment by Kummo33** [comment]
No I meant before I had the new wizbang decks IN the deck tracker, not in the game. I deleted all the decks in my deck tracker to refresh the stats but the new wizbang decks didn't come back for some reason on the deck tracker. So it was causing the card disappearing issue.

I couldn't get it regenerate unlike prior to Uldum where it regenerates instantly. Also all the times I mentioned was after the expansion was released, not before.

Though I managed to find a work around today. I logged into another account on another region that didn't have a wizbang, then afterwards when I logged back into my wizbang account, the tracker managed to regenerate the new wizbang decklist again. So it's all good now.
